roles/deprovision-keycloak-apb/tasks/cleanup-keycloak.yml

```diff
@@ -1,6 +1,7 @@
 ---
 - set_fact:
     kc_namespace_configmap: "oc get configmaps/{{ keycloak_configmap_name }} --namespace={{ namespace }}"
+    kc_namespace_secret: "oc get secrets --namespace={{ namespace }}"
```

Comment:
can we create a follow up Jira to change this once the fix for https://github.com/openshift/ansible-service-broker/issues/818 is in place

Reply:
Dont wanna create JIRA. Better to track #818 and when its merged then we can create ticket about changes. I added TODO with issue link